CODEOWNERS

```diff
@@ -0,0 +1,110 @@
+# ==============================================================================
+#                          Bitcoin Core CODEOWNERS
+# ==============================================================================
+
+# Configuration of code ownership and review approvals for the bitcoin/bitcoin
+# repo.
+
+# Order is important; the last matching pattern takes the most precedence.
+# More info on how this file works can be found at:
+# https://help.github.com/articles/about-codeowners/
+
+# This file is called CODEOWNERS because it is a magic file for GitHub to
+# automatically suggest reviewers. In this project's case, the names below
+# should be thought of as code reviewers rather than owners. Regular
+# contributors are free to add their names to specific directories or files
```

Comment:
Should reviewers who wish to be added/removed make a PR for this? Should the process be described?

Reply:
Perhaps a gist or devwiki could be used to request being added/removed, and after a regular time period the changes to this file can be batched.

Reply:
I think anyone who has reviewed more than 5 pull requests AND has been active reviewing for more than 14 days should be allowed to add themselves in a pull request. Such a pull request doesn't need review and can be merged right away when the conditions have been fulfilled. (Exact conditions are up for debate)